docs/concepts/configuration/assign-pod-node.md

```diff
@@ -190,13 +190,7 @@ flavor and the `preferredDuringSchedulingIgnoredDuringExecution` flavor.
 
 As with node affinity, the legal operators for pod affinity and anti-affinity are `In`, `NotIn`, `Exists`, `DoesNotExist`, `Gt`, `Lt`.
 
-In principle, the `topologyKey` can be any legal label value. However,
-for performance reasons, only a limited set of topology keys are allowed.
-By default the allowed topology keys are:
-
-* `kubernetes.io/hostname`
-* `failure-domain.beta.kubernetes.io/zone`
-* `failure-domain.beta.kubernetes.io/region`
+The `topologyKey` can be any legal label value.
```

Comment:
This seems not accurate, as for hard anti-affinity, if `LimitPodHardAntiAffinityTopology` enabled, then the `topologyKey` must be LabelHostname, so how about describe the behavior with some items:

1) For hard pod anti-affinity, without the admission controller `LimitPodHardAntiAffinityTopology` enabled, `topologyKey` can be any non-empty value.
2) For hard pod anti-affinity, with the admission controller `LimitPodHardAntiAffinityTopology` enabled, `topologyKey` must be `kubernetes.io/hostname`.
3) For soft pod anti-affiniy, `topologyKey` can be any value when specified. If the `topologyKey` was not specified, then it will use default value as `failure-domain.beta.kubernetes.io/zone`.
4) For the non above cases, the `topologyKey` can be any non-empty value.

Comments? Thanks.